### PR TITLE
Fixing accessibility issue on menu bar

### PR DESCRIPTION
**Pull request by ahmad-shahid**

### CODE REVIEW

File: public/gcintranet/ajax/custommenu-eng.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <h2>Topics menu</h2>
   <div class="row">
     <ul class="list-inline menu" role="menubar">
-      <li><a href="#" class="item">Menu item 1</a>
+      <li role="presentation"><a href="#" class="item">Menu item 1</a>
```
Comment (plup2) [inline public/gcintranet/ajax/custommenu-eng.html:10]: As discussed, should it be `role="None"` ?

File: public/gcintranet/ajax/custommenu-eng.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
...
         <ul class="sm list-unstyled" id="project" role="menu">
           <li><a href="#">Menu item 1-1</a></li>
```
Comment (plup2) [inline public/gcintranet/ajax/custommenu-eng.html:12]: As discussed, `role="None"` as well for submenu items?

File: test/specs/appTop.js [hunk 3]
```diff
@@ -745,3 +749,8 @@ async function accessibility(theme, lang) {
     await appTopPage.open(theme, lang);
     await runAccessbilityTest();
 }
+
+async function accessibilityExternalMenu(theme, lang) {
```
Comment (plup2) [inline test/specs/appTop.js:753]: We don't need a separate function here, we should change the `accessibility` function to be `async function accessibility(theme, lang, classifier = '')`

**Review by plup2 — CHANGES_REQUESTED** [review]

File: public/gcintranet/ajax/custommenu-eng.html (hunk 1 above)
Comment (nschonni) [reply public/gcintranet/ajax/custommenu-eng.html:10]: https://w3c.github.io/aria-practices/examples/menubar/menubar-navigation.html

File: public/gcintranet/ajax/custommenu-eng.html (hunk 2 above)
Comment (ahmad-shahid) [reply public/gcintranet/ajax/custommenu-eng.html]: Role has been updated

File: test/specs/appTop.js (hunk 3 above)
Comment (ahmad-shahid) [reply test/specs/appTop.js]: done

File: public/gcintranet/ajax/custommenu-eng.html (hunk 1 above)
Comment (ahmad-shahid) [reply public/gcintranet/ajax/custommenu-eng.html]: Role has been updated

**Review by plup2 — APPROVED** [review]

**Pull request merged**